x-pack/plugins/security_solution/server/lib/types.ts

```diff
@@ -64,6 +65,12 @@ export interface TotalValue {
   relation: string;
 }
 
+export interface BaseHit<T> {
```

Comment:
These types are cribbed from #77419, FYI. It doesn't look like they'll conflict but we'll see 🤞